pa11ycrawler/cmdline.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python
+"""
+The command line interface for pa11ycrawler.
+"""
+import argparse
+
+from textwrap import dedent
+
+from scrapy.crawler import CrawlerProcess
+
+from pa11ycrawler.reporter import HtmlReporter
+from pa11ycrawler.settings_helpers import (
+    is_pa11y_setting,
+    is_pa11ycrawler_setting,
+    get_default_settings,
+)
+from pa11ycrawler.spiders.pa11y_spider import Pa11ySpider
+
+# ------------ Main CLI ------------
+
+
+def cli():
+    """
+    Returns the main CLI parser.
+    """
+    parser = argparse.ArgumentParser(prog='pa11ycrawler')
+    subparsers = parser.add_subparsers(title='commands')
+    run_cli(subparsers)
+    json_to_html_cli(subparsers)
+    return parser
+
+# ------------ "run" CLI -----------
+
+
+def run_cli(subparsers):
+    """
+    Creates the parser for the `run` command
+    """
+    run_usage = dedent('''pa11ycrawler run [options] url [url ...]
+
+    ---------------------------------------------------------------
+    NOTE: This only documents a haldful of the possible options.
+    More documentation can be found for the following categories:
+
+    CUSTOM SCRAPY SETTINGS:
+    ** Implemented for pa11ycrawler. (e.g. Not found in Scrapy Docs.)
+    ** All options documented below.
+    ** Most likely to need to be changed.
+
+    PA11Y SETTINGS:
+    ** All options documented below. More details can be found at
+       https://github.com/springernature/pa11y#command-line-interface
+
+    STANDARD SCRAPY SETTINGS:
+    ** http://doc.scrapy.org/en/latest/topics/settings.html
+    ** These can be set using typical flag formatting. For example,
+       --depth-limit=2 will set the DEPTH_LIMIT setting to 2.
+    ---------------------------------------------------------------
+    ''')
+
+    parser_run = subparsers.add_parser(
+        'run',
+        usage=run_usage,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        help="Run the crawler, checking accessibility along the way!"
+    )
+    parser_run.set_defaults(func=run)
+    parser_run.add_argument(
+        'PA11YCRAWLER_START_URLS',
+        metavar='url',
+        nargs='+',
```

Comment:
Yea... `nargs='+'` translates to "1 or more".   I think `nargs='*'` would make it "0 or more".